sled-agent/src/instance.rs

```diff
@@ -1092,13 +1150,48 @@ fn propolis_error_code(
 }
 
 /// Describes a single Propolis server that incarnates a specific instance.
+#[derive(Clone)]
```

Comment:
The intent behind deriving `Clone` for this is to allow the instance's handle to be cloned  into tasks spawned by the `InstanceManager` for the functions that need to send multiple requests to the instance (or requests to multiple instances) rather than just passing in the `tx` from the original request (e.g. `use_only_these_disks`). I didn't actually do that in this PR but intend to do so in a follow-up.

Reply:
as a *tiny* nitpick I might punt on deriving clone until that PR, because it doesn't seem necessary in this one.

(but also, not a big deal if this is a pain in the butt)